docs: document en-US runtime loading skip for Grafana 13.1.0 (#2645)
grafana/plugin-tools: docusaurus/docs/shared/plugin-internationalization-shared.md

## Determine the text to translate

After you've configured your plugin for translation, you can proceed to mark up the language strings you want to translate. Each translatable string is assigned a unique key that ends up in each translation file under `locales/<locale>/<plugin id>.json`.

:::note

> The string you pass as the default (the second argument of `t()` or the children of `<Trans>`) is what Grafana renders for `en-US`. Starting with Grafana 13.1.0, translation resources for `en-US` are **not** loaded at runtime. They're read directly from your source code. This keeps both the plugin and Grafana from fetching a translation file the bundle already contains, saving hundreds of kilobytes on first paint. The `src/locales/en-US/<plugin-id>.json` file is still generated by `i18n-extract` and used as the source of truth for translators, but it is not fetched by the browser. Keep your in-source defaults meaningful and up to date.

Reviewer: ```suggestion
The string you pass as the default (the second argument of `t()` or the children of `<Trans>`) is what Grafana renders for `en-US`. Starting with Grafana 13.1.0, translation resources for `en-US` are **not** loaded at runtime. Instead, they're read directly from your source code, so both the plugin and Grafana don't need to fetch a translation file the bundle already contains, saving hundreds of kilobytes on first paint. The `src/locales/en-US/<plugin-id>.json` file is still generated by `i18n-extract` and used as the source of truth for translators, but it's not fetched by the browser. Keep your in-source defaults meaningful and up to date.
```
